hack/machineset.sh

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env bash
+
+# machineset.sh - create a Windows MachineSet for Windows Machine Config Operator
+# Creates a MachineSet.yaml file and apply/delete the MachineSet if optional action is provided
+#
+# USAGE
+#    machineset.sh
+# OPTIONS
+#    $1      Action       (Optional) apply/delete the MachineSet
+# PREREQUISITES
+#    oc                   to fetch cluster info and apply/delete MachineSets on the cluster(cluster should be logged in)
+#    aws                  to fetch Windows AMI id for AWS platform (only required for clusters running on AWS)
+
+function error-exit() {
+    echo "Error: $*" >&2
+    exit 1
+}
+
+ACTION=$1
+
+# get_spec returns the template yaml common for all cloud providers
+get_spec(){
+
+local infraID=$1
+local az=$2
+local provider=$3
+
+machineSetName="$infraID"-windows-worker-"$az"
+if [ "$provider" = "azure" ]; then
+  machineSetName="winworker"
```

Comment:
a comment on why this is necessary would be great

Reply:
done